ocaml/xapi/xapi_pusb.ml

```diff
@@ -29,8 +29,14 @@ let create ~__context ~uSB_group ~host ~other_config ~path
   debug "PUSB ref='%s' created" (Ref.string_of pusb);
   pusb
 
+let host_ref : [`host] Ref.t ref = ref Ref.null
+
 let scan_start ~__context usbs =
   let host = Helpers.get_localhost ~__context in
+  if !host_ref <> host then
```

Comment:
we don't need this.

Reply:
how we pass the host here? just 
let scan_start ~__context usbs host =
?

Reply:
we don't need to pass it at all. It's always localhost.

Reply:
updated.